rerun_py/rerun_sdk/rerun2/components/class_id_ext.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+__all__ = ["ClassIdArrayExt"]
+
+from typing import Any, Sequence, Type
+
+import numpy as np
+import pyarrow as pa
+
+
+class ClassIdArrayExt:
+    @staticmethod
+    def _from_similar(data: Any | None, *, mono: type, mono_aliases: Type, many: type, many_aliases: Type, arrow: type):
+        if isinstance(data, Sequence) and (len(data) > 0 and isinstance(data[0], mono)):
+            array = np.asarray([class_id.id for class_id in data], np.uint16)
+        else:
+            array = np.require(np.asarray(data), np.uint16).flatten()
```

Comment:
Why not set the `dtype` on `asarray` instead of doing an additional call?
(applies to several places)